core/src/main/scala/org/apache/spark/api/python/PythonRDD.scala

```diff
@@ -425,6 +425,11 @@ private[spark] object PythonRDD extends Logging {
     iter.foreach(write)
   }
 
+  /** Create an RDD that has no partitions or elements. */
+  def emptyRDD[T](sc: JavaSparkContext): JavaRDD[T] = {
```

Comment:
I think this is just bringing it to parity with the scala SparkContext API, which we currently also only use in tests. This is probably fine.